.github/workflows/jetbrains-updates-template.yml

```diff
@@ -59,12 +59,24 @@ jobs:
             This PR updates the ${{ inputs.productName }} IDE image to the latest release version.
 
             ## How to test
-            Go to the preferences and select ${{ inputs.productName }} as desktop IDE. Start a workspace and verify that the new IDE version works.
+            1. Open the preview environment generated for this branch
+            2. Choose ${{ inputs.productName }} as your preferred editor
+            3. Start a workspace using this repository: `https://github.com/gitpod-io/spring-petclinic`
```

Comment:
it is only true for IntelliJ, we have automated PRs for all JB supported products

Reply:
@andreafalzetti You can make it(repo) a variable and input it from `jetbrains-updates.yml` like others

Reply:
Actually it bring a valid point that for each product we should have a repo with different configuration enabled like preconfigured plugins and prebuilds. Maybe for another PR to add such configurations though.

Reply:
@akosyakov thanks for clarifying, I've updated the wording